spec/models/user_spec.rb

```diff
@@ -33,10 +35,10 @@
     it { should_not be_valid }
   end
 
-  describe "when email with mixed case" do
-    let(:mixed_case_email) { "Foo@Example.Com" }
+  describe 'when email with mixed case' do
+    let(:mixed_case_email) { 'Foo@Example.Com' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.